### ISSUE TITLE
Clicking on a info window does not bring focus on the referenced feature on some devices

### ISSUE THREAD

**Issue by ekigamba** [onaio/kujaku#129]
This has been note on the following devices:

- [ ] Android API 26
- [ ] Android API 25

This seems to happens on some features for some clicks, it doesn't happen always and could be related to the touch not being on a feature

**Comment by ekigamba** [comment]
@craigappl Can you be able to reproduce this issue on your tablet?

**Comment by craigappl** [comment]
@ekigamba No, I can not reproduce this on the tablet. When I touch a feature on my tablet, the list of cards pulls up and it quickly shifts to the card for that feature. Note that the icon doesn't change on the map to signal which card is linked to which feature I touched.

**Comment by ekigamba** [comment]
> Note that the icon doesn't change on the map to signal which card is linked to which feature I touched.

We have this problem and are looking at enlarging the feature to make it bigger than the others, I am not sure if this will suffice for the circle layer we have or we have to come up with something better for showing the selected feature
